src/OSSGadget.sln

```diff
@@ -30,6 +30,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "oss-tests", "oss-tests\oss-
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "oss-metadata", "oss-metadata\oss-metadata.csproj", "{7E9A489A-B502-4622-A34C-EF507E18DFEB}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "oss-find-squats", "oss-find-squats\oss-find-squats.csproj", "{23D17E08-9E95-4448-B75D-BABAF1F3C854}"
+EndProject
```

Comment:
.\oss-find-squats.exe pkg:npm/express gave me a bunch of error messages. Is this too verbose, especially when you add a text option? 

Checking �express.
[33mError enumerating NPM package: Response status code does not indicate success: 404 (Not Found).[0m
Checking �express.
[33mError enumerating NPM package: Response status code does not indicate success: 404 (Not Found).[0m
Checking �express.
[33mError enumerating NPM package: Response status code does not indicate success: 404 (Not Found).[0m
Checking �express.
[33mError enumerating NPM package: Response status code does not indicate success: 404 (Not Found).[0m
Checking �express.
[33mError enumerating NPM package: Response status code does not indicate success: 404 (Not Found).[0m
Checking �express.
[33mError enumerating NPM package: Response status code does not indicate success: 404 (Not Found).[0m
Checking �express.
[33mError enumerating NPM package: Response status code does not indicate success: 404 (Not Found).[0m
Checking �express.
[33mError enumerating NPM package: Response status code does not indicate success: 404 (Not Found).[0m
Checking �express.
[33mError enumerating NPM package: Response status code does not indicate success: 404 (Not Found).[0m

Reply:
The 404s come from the various project managers (i think they should not but didn’t want to change it unilaterally in case other projects depend on that output)

just 404s was meaningless so I added the names. 

I don’t want either.

Reply:
@scovetta @jacobmsft are you find with me removing these logger.warns/errors when getting a 404?

Reply:
How about make it a Trace call instead so we capture it somewhere. Just thinking about the other scenarios -- want to make sure that `oss-download pkg:npm/non-existent` provides an error back to the user somehow that the package wasn't found.

Reply:
Trace sounds good to me. For any error downloading the package, I think emitting and catching an exception might be better, instead of console writes from a shared library.

Reply:
I’ll do trace and rethrow for enumerate versions.